<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWindow</class>
 <widget class="QDialog" name="addWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="addWindowTitle">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="name">
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="phoneNumber">
        <property name="text">
         <string>phone number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneNumberInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="email">
        <property name="text">
         <string>email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailInput"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="addContactBtn">
      <property name="text">
       <string>create contact</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
